### PR TITLE
Parallel travis

### PR DESCRIPTION
**Pull request by LocoDelAssembly**
Changes:
* Parallelized test using five workers
* Clear assets and VCR caches on CRON event.
* Conditional assets precompilation based on set of specs the worker runs.
* Configured coveralls to merge all workers into single report
* Using native headless Firefox rather than emulated screen buffer.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -53,23 +54,18 @@ install:
...
-  - ls -alh public/packs-test 
 script:
-  - bundle exec rake assets:precompile
-# - bundle exec parallel_test spec/ -n 6 --only-group $TEST_GROUP --group-by filesize --type rspec
-  - bundle exec rspec
+  - .travis/spec_runner.sh
 notifications:
   email:
     recipients:
       - diapriid@gmail.com
```
Comment (mjy) [inline .travis.yml]: Should we add others here? @LocoDelAssembly?

File: .travis/spec_runner.sh [hunk 2]
```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
```
Comment (mjy) [inline .travis/spec_runner.sh]: @LocoDelAssembly Is this code based on existing code somewhere "out there"?  Please reference in comments if so.  If not- very nice.

File: .travis.yml (hunk 1 above)
Comment (LocoDelAssembly) [reply .travis.yml]: @mjy I guess I could add myself there.
```suggestion
      - ellocodelassembler@gmail.com
```

File: .travis/spec_runner.sh (hunk 2 above)
Comment (LocoDelAssembly) [reply .travis/spec_runner.sh]: @mjy I believe that other searching for the correct bash syntax it was all mine.

**Pull request merged**